tests/nightly/unity/task.yaml

```diff
@@ -5,19 +5,14 @@ environment:
 
 systems: [ubuntu-16.04-64]
 
-execute:
-    # FIXME Test is broken. Downloads HUNDREDS and HUNDREDS of packages,
-    # and asks for input on configuration file diffs.
```

Comment:
mm `DEBCONF_FRONTEND=noninteractive` would prevent the modified config file error from surfacing, right? I would like to understand the problem (if it's still there) instead of ignoring it.